content/en/docs/components/pipelines/v1/installation/localcluster-deployment.md

```diff
@@ -318,11 +314,15 @@ Executors](https://argoproj.github.io/argo-workflows/workflow-executors/) and re
 1. To deploy the Kubeflow Pipelines, run the following commands:
 
     ```shell
-    # env/platform-agnostic-pns hasn't been publically released, so you will install it from master
-    export PIPELINE_VERSION={{% pipelines/latest-version %}}
-    kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=$PIPELINE_VERSION"
-    kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
-    kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/env/platform-agnostic-pns?ref=$PIPELINE_VERSION"
+
+    git clone https://github.com/kubeflow/pipelines.git 
+    cd pipelines/manifests/kustomize
```

Comment:
Can you use the script that doesn't require git clone the repository? It includes downloading the whole codebase, which is not necessary.